.coderabbit.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://storage.googleapis.com/coderabbit_public_assets/schema.v2.json
+language: en
+reviews:
+  profile: chill
+  request_changes_workflow: false # Do not approve after comments resolved
+  high_level_summary: true
+  review_status: true
+  collapse_walkthrough: true
+  changed_files_summary: true
+  sequence_diagrams: true
```

Comment:
Let's turn this off, it hasn't ever been helpful to me.

Reply:
Resolved in [4f4244d](https://github.com/PrairieLearn/PrairieLearn/pull/11337/commits/4f4244dabff9190d6411769f7b0bdfa6f2e8e999).